MarsBaseBuilder/Measures.fs

```diff
@@ -0,0 +1,22 @@
+﻿module MarsBaseBuilder.Measures
+
+[<Measure>]
+type gameUnit
```

Comment:
Interference with type GameUnit. If it's unit of position maybe better use real SI units like `km`, `m`.